src/web/app/pages-admin/admin-search-page/admin-search-page.component.html

```diff
@@ -92,7 +92,7 @@
         </tr>
       </thead>
       <tbody>
-        <ng-container *ngFor="let student of students">
+        <ng-container *ngFor="let student of students;">
```

Comment:
Unrelated change; although this doesn't affect correctness, try not to have this kind of stuff.